app/src/main/AndroidManifest.xml

```diff
@@ -48,8 +48,8 @@
                 <category android:name="android.intent.category.BROWSABLE"/>
 
                 <data
-                    android:host="login"
-                    android:scheme="fasthub"/>
+                    android:host="fast-for-github.firebaseapp.com"
```

Comment:
Jedi this shouldn't be modified, the whole manifest, I have changed the scheme url for login, but yet this commit brings it back and will break if I release.